lib/jekyll-feed/generator.rb

```diff
@@ -8,6 +8,8 @@ class Generator < Jekyll::Generator
     # Main plugin action, called by Jekyll-core
     def generate(site)
       @site = site
+      return if config["skip_development"] == true && Jekyll.env == "development"
```

Comment:
Let's log in this case so it's clear to the user what's happening. Something like:

```ruby
Jekyll.logger.info "Jekyll Feed:", "Skipping feed generation in development"
```